### DISCUSSION TITLE
org.eclipse.glsp.server.diagram.DiagramConfiguration access specifier issue

### DISCUSSION THREAD

**Discussion by sathya-1994** [eclipse-glsp/glsp#625] [General] [answered]
org.eclipse.glsp.server.diagram.DiagramConfiguration.animatedUpdate() method will return always true. But my requirement is to avoid animation of diagrams while update.

I cannot override animatedUpdate() method in the child class as it is a default method present in the interface.

Any way we can configure animation flag?

**Comment by tortmayr** [comment, marked as answer]
You can overwrite default methods of interfaces just as any other interface method.
The following code in your `DiagramConfiguration` class should work just fine.
```java
  @Override
   public boolean animatedUpdate() {
      return false;
   }
```

**Comment by sathya-1994** [comment]
@tortmayr ,

Sorry, earlier I was using default as access specifier

 **default** boolean animatedUpdate() {
		return false;
}

My issue is resolved.
